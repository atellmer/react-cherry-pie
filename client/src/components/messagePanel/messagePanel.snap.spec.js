import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmMessagePanel from './index';


const props = {
  changeMessagePanelHeight: () => {}
};

test('Jest: TmMessagePanel (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();

  const result = shallowRenderer.render(<TmMessagePanel {...props}/>);
  expect(result).toMatchSnapshot();
});
