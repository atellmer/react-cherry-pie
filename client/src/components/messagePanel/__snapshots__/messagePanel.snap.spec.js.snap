exports[`test Jest: TmMessagePanel (Snapshot) 1`] = `
<form
  onSubmit={[Function]}>
  <Flex
    align="center"
    className={undefined}>
    <Box>
      <IconButton
        disableTouchRipple={false}
        disabled={false}
        iconStyle={Object {}}
        tooltipPosition="bottom-center"
        touch={false}>
        <SocialSentimentSatisfied
          color="#7f7f7f" />
      </IconButton>
    </Box>
    <Box
      flexAuto={true}>
      <TmTextarea
        onChangeInput={[Function]}
        onHeightChange={[Function]}
        value="" />
    </Box>
    <Box>
      <IconButton
        disableTouchRipple={false}
        disabled={false}
        iconStyle={Object {}}
        tooltipPosition="bottom-center"
        touch={false}
        type="submit">
        <ContentSend
          color="#9a85cd" />
      </IconButton>
    </Box>
  </Flex>
</form>
`;
